shared/fs/common/pie-slice.tsx

```diff
@@ -76,12 +87,15 @@ const styles = Styles.styleSheetCreate({
     ...stylePieWhole,
     left: 0,
   },
-  wholeGrey: {
-    ...stylePieWhole,
+  unfilledNegative: {
+    backgroundColor: Styles.globalColors.blueDark,
```

Comment:
I think this blue might be a little too close to the filled blue when it's empty, especially on a non-blue background (there are red and green too). Cécile suggested `black_05` or similar.

Reply:
Transparent black doesn't work because it doesn't cover the unfilled sections.

Reply:
Oh, I didn't realize 05 meant transparent. Maybe a black that is slightly gray? I'm ok with the current color though if there isn't an obvious alternative.

Reply:
I think we should keep it as is right now, and then if we ever need it on a red/green dark background, we can refactor again to explicitly mention background colors instead of positive/negative.